src/main/java/com/github/tomakehurst/wiremock/common/StreamSources.java

```diff
@@ -63,4 +63,16 @@ public InputStream getStream() {
   public static InputStreamSource empty() {
     return forBytes(new byte[0]);
   }
+
+  public static String getStringFromInputStream(InputStream inputStream) {
+    try {
+      StringBuilder sb = new StringBuilder();
+      for (int ch; (ch = inputStream.read()) != -1; ) {
+        sb.append((char) ch);
+      }
+      return sb.toString();
+    } catch (Exception ignored) {
+      return null;
+    }
+  }
```

Comment:
This is only used in tests - I wonder if we could move it out of the production codebase?

Concerns otherwise:
* Looks like it is unbuffered, which would have performance implications if used in the production code.
* Swallows any exception - `IOUtils.toString(InputStream, Charset)` propagates it, so it could just throw it

Reply:
removed